addons/account/models/account_journal_dashboard.py

```diff
@@ -814,6 +820,7 @@ def open_action(self):
                 action['domain'] = [(domain_type_field, 'in', ('in_invoice', 'in_refund', 'in_receipt', 'entry'))]
 
         action['domain'] = (action['domain'] or []) + [('journal_id', '=', self.id)]
+        action['name'] = _('%(action)s for journal %(journal)s', action=action['name'], journal=self.name)
```

Comment:
Useless IMO Isn't it clear enough from the linked you just clicked on the dashboard ?  I can discuss that with a PO if you really want it.